VestingInstance.sol

```diff
@@ -0,0 +1,273 @@
+pragma solidity ^0.5.3;
+
+import "openzeppelin-solidity/contracts/ownership/Ownable.sol";
+import "openzeppelin-solidity/contracts/math/SafeMath.sol";
+import "../common/UsingRegistry.sol";
+
+contract VestingSchedule is Ownable, UsingRegistry {
```

Comment:
What would be the owner privileges be in this contract?

Reply:
none that I have thought of, tbh, hence remvoed